### PR TITLE
new file:   packages/acpc/acpc.2.0.0/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**
compiles with ocaml-5 and dune (instead of obuild previously)

### CODE REVIEW

File: packages/acpc/acpc.2.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+depends: [
+  "base-unix"
+  "batteries"
+  "bitv"
+  "conf-autoconf"
+  "conf-gnuplot"
+  "dolog"
+  "dune" {>= "1.11"}
+  "ocaml" {>= "5.0.0"}
+  "parany"
```
Comment (kit-ty-kate) [inline packages/acpc/acpc.2.0.0/opam:19]: Does not expect `Parmap.parmap` to have a `~ncores` argument
```suggestion
  "parany" {>= "11"}
```

**Comment by UnixJunkie** [comment]
Can't the solver find a proper parany version?
There are only few which are OK with ocaml>=5.0.0.

File: packages/acpc/acpc.2.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+name: "acpc"
```
Comment (mseri) [inline packages/acpc/acpc.2.0.0/opam:2]:
```suggestion
```

**Comment by mseri** [comment]
The remaining failures are unrelated

**Pull request merged**